fenix/.buildconfig.yml

```diff
@@ -62,6 +64,7 @@ projects:
     - lib-publicsuffixlist
     - lib-push-firebase
     - lib-state
+    - mozilla-lint-rules
```

Comment:
@csadilek I see the decision task is busted because of this line. I'm not sure why gradle reports it on Fenix not anywhere else:

```
./gradlew --parallel dependencies :app:dependencies
[...]
lintChecks - Configuration to apply external lint check jar
\--- project :mozilla-lint-rules
```

I guess it's related to this line but I'm unsure what to do: 

https://github.com/mozilla-mobile/firefox-android/blob/aa3adc0e486c55efa18f26e4a29615f2ba4aac5a/fenix/settings.gradle#L29

Reply:
@JohanLorenzo This is a real project in Fenix from https://github.com/mozilla-mobile/firefox-android/tree/main/fenix/mozilla-lint-rules. We will have to handle this in our script and ignore it.

Reply:
I don't think we can do this using Gradle, so it would have to be filtered out in our parser, or alternatively, we can add the project explicitly so it's known to avoid the `KeyError`?

Reply:
Yeah, let's filter out this gradle project for now. It's not ideal but at least we'll make progress on the rest transitive dependencies.